submissions/mayowaShoyinka/htmlForms/contact.html

```diff
@@ -1,62 +1,77 @@
 <!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="utf-8" >
-        <title>Contact-Us</title>
-    </head>
-    <body >
-
-
-        <nav>
-        <div>
-            <img src="../html/logo.png"/>
-            <ul>
-                <li><a href="#">Home</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="../html/index.html">Catlog</a></li>
-                <li><a href="#">Blog</a></li>
-                <li><a href="contact.html">Contact</a></li>
-                <li><a href="#">Catlog</a></li>
-                <li><a href="#">Blog</a></li>
-                <li><a href="#">Contact</a></li>
-
-            </ul>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<title> Contact || </title>
+    <link rel="stylesheet" href="../css/contact.css">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&family=Raleway:ital,wght@1,300&family=Roboto:ital,wght@1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../html/img/font/flaticon.css">
```

Comment:
Are you using this @MayowaShoyinka ? I don't see this in your PR.

Reply:
can i close this PR and upload a new one

Reply:
Why?

Reply:
have made changes, but you said you cant see it

the one you talked about: i already closed the PR

Reply:
please can you check it @ToluAkin

Reply:
@MayowaShoyinka It is still there.